src/devices/src/virtio/net/mod.rs

```diff
@@ -15,9 +15,11 @@ pub const TX_INDEX: usize = 1;
 
 pub mod device;
 pub mod event_handler;
+pub mod persist;
 
 pub use self::device::Net;
 pub use self::event_handler::*;
+pub use self::persist::*;
```

Comment:
nit: I wouldn't `pub use persist::*` and keep it consistent with the other devices. Users could explicitly `use devices::net::persist::{X, Y}` since persist is a public module.

Reply:
Done.